utils/dchemigen/readpng.h

```diff
@@ -0,0 +1,40 @@
+/* typedef unsigned int uint32;
```

Comment:
Can you make sure this file has a KOS copyright header similar to readpng.c and the rest?

Reply:
This is missing header where I lifted it from (dcbumpgen).